src/Tools/dotnet-trace/CommandLine/Commands/CollectCommand.cs

```diff
@@ -184,11 +184,13 @@ private static async Task<int> Collect(CancellationToken ct, IConsole console, i
                         Action printStatus = () =>
                         {
                             if (!Console.IsOutputRedirected)
+                            {
                                 rewriter?.RewriteConsoleLine();
+                                fileInfo.Refresh();
+                                Console.Out.WriteLine($"[{stopwatch.Elapsed.ToString(@"dd\:hh\:mm\:ss")}]\tRecording trace {GetSize(fileInfo.Length)}");
+                                Console.Out.WriteLine("Press <Enter> or <Ctrl+C> to exit...");
+                            }
```

Comment:
If output is redirected, do we want to print a single status message at the end indicating the trace size and duration?

Reply:
Good idea @josalem: I will implement it in a new PR in 2 weeks (after my vacations ;^)